suppressible-error-prone/build.gradle

```diff
@@ -21,7 +21,9 @@ moduleJvmArgs {
             'jdk.compiler/com.sun.tools.javac.comp',
             'jdk.compiler/com.sun.tools.javac.file',
             'jdk.compiler/com.sun.tools.javac.main',
+            'jdk.compiler/com.sun.tools.javac.model',
             'jdk.compiler/com.sun.tools.javac.parser',
+            'jdk.compiler/com.sun.tools.javac.processing',
```

Comment:
Checking out the PR locally, it seems like we might need to add these to `gradle.properties` too?

Reply:
Oh nevermind, I had meant to remove this comment. I just needed to have intellij somehow add it to _something_ because it didn't find the classes